src/Database/Driver/PDODriverTrait.php

```diff
@@ -94,9 +94,8 @@ public function isConnected()
                 $connected = false;
             }
         }
-        $this->connected = !empty($connected);
 
-        return $this->connected;
+        return (bool)$connected;
```

Comment:
You changed the behavior of this method. The `connected` property is no longer assigned.